Add docs for <ResetButton /> (#892)
zeit/docs: pages/docs/integrations/v2/integrations-docs-mdx/component-catalog/input-components.mdx

  "key2": "value2"
}
```

### ResetButton

> The ResetButton component allows the user to reset the values in the `<Input />` components.

Reviewer: ```suggestion
The ResetButton component allows the user to reset the values of `<Input />` components.
```

> It will reset it on the client side, which means that no server side request will be made.

Reviewer: ```suggestion
Values are reset on the client side, which means that no server side request will be made.
```

```jsx
<ResetButton targets="username, password">Reset</ResetButton>
```

The ResetButton component supports the following attribute:

- `targets` - comma separated values of the `<Input />` names. If the attribute is not defined it will reset the values of all `<Input />` components.

Below is an example showing the component's usage:

```jsx
<Page>
  <Input type="text" name="username" />
  <Input type="text" name="password" />
  <Input type="text" name="age" />

  <Button action="submit">Submit</Button>
  <ResetButton targets="username, password">Reset</ResetButton>

> </Page>

Reviewer: ```suggestion
In the above example, the `username` and `password` fields will be reset, whilst the `age` field will remain unchanged.

</Page>
```
